### PR TITLE
Completed day 4 problems number 1 - 11.

### PR DESCRIPTION
**Pull request by thisisrai**

### CODE REVIEW

File: sql/07-query-friends-gabriel.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+SELECT id FROM student.student WHERE name = 'Gabriel';
+
+/* Picks student ids of friends of Gabriel from student.friends table from id1 column*/
+SELECT id1 FROM student.friends WHERE id2 IN (SELECT id FROM student.student WHERE name = 'Gabriel');
+
+/* Picks student ids of friends of Gabriel from student.friends table from id2 column*/
+SELECT id2 FROM student.friends WHERE id1 IN (SELECT id FROM student.student WHERE name = 'Gabriel');
+
+/*Picks names from student.student table from given ids from line 10 and 12*/
+SELECT name from student.student WHERE id IN (SELECT id1 FROM student.friends WHERE id2 IN (SELECT id FROM student.student WHERE name = 'Gabriel')) OR id IN (SELECT id2 FROM student.friends WHERE id1 IN (SELECT id FROM student.student WHERE name = 'Gabriel'));
```
Comment (shakalee14) [inline sql/07-query-friends-gabriel.sql]: can we make some of these multi-line for readability?

File: sql/01-create-table-student.sql [hunk 2]
```diff
@@ -1 +1,5 @@
-CREATE TABLE students(id INT, name TEXT, grade INT);
+psql sqlexercises; -- To get into database
+
+CREATE SCHEMA student; --To create the shcema for the project.
```
Comment (shakalee14) [inline sql/01-create-table-student.sql]: i dont believe creating a schema is necessary.
https://www.postgresql.org/docs/9.3/static/sql-createschema.html

File: sql/04-load-table-student.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+INSERT INTO student.student(id, name, grade) VALUES (1510, 'Jordan', 9),
```
Comment (shakalee14) [inline sql/04-load-table-student.sql:2]: why are you doing student.student? is that for the schema?

File: sql/05-load-table-friend.sql [hunk 4]
```diff
@@ -0,0 +1,21 @@
+INSERT INTO student.friends(id1, id2) VALUES
```
Comment (shakalee14) [inline sql/05-load-table-friend.sql:1]: i think it just assumes your user name if you don't create a schema. i think schema's have to deal with permission issues.

File: sql/08-query-likes-grade-two-or-more.sql [hunk 5]
```diff
@@ -0,0 +1,24 @@
+/* **08:** For every student who likes someone 2 or more grades younger than themselves, return that student's name and grade, and the name and grade of the student they like.
```
Comment (shakalee14) [inline sql/08-query-likes-grade-two-or-more.sql:1]: great!

**Review by shakalee14 — COMMENTED** [review]
few comments

**Comment by thisisrai** [comment]
I misread the readme and made a schema named student and a table is actually also named student.

**Pull request merged**